packages/boxel-ui/addon/components/modal/index.gts

```diff
@@ -119,6 +121,17 @@ export default class Modal extends Component<Signature> {
         --boxel-modal-max-width: 65rem; /* 1040px */
       }
 
+      .fullScreen {
```

Comment:
I suggest renaming to `full-screen` for consistency with other css classes

Reply:
looks like templates are totally cool with this:
![image](https://github.com/cardstack/boxel/assets/61075/0778053d-62f2-4819-be32-b95d4d0622d7)